src/type_checker/resources/std_lib/collection.py

```diff
@@ -0,0 +1,21 @@
+from typing import TypeVar, Generic
+
+class set(Generic[T]):
+    def __init__(self): pass
+    def __iter__(self) -> set_iterator[T]: pass
+
+    def __bool__(self) -> bool: pass
+
+class set_iterator(Generic[T]):
+    def __init__(self): pass
+    def __next__(self) -> T: pass
+
+class list(Generic[T]):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'list'](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4163821)